### PR TITLE
Add env variables that tt CLI uses

### LINKED ISSUE #3828: Document environment variables supported by tt
**Product:** tt
**Since:** 2.2.0
**Root document:** tt commands reference pages: https://www.tarantool.io/en/doc/latest/reference/tooling/tt_cli/commands/

**SME:** @ oleg-jukovec

# Details

* `tt connect`: `TT_CLI_USERNAME` and `TT_CLI_PASSWORD` - instance connection credentials (already documented)
* `tt cluster`: : `TT_CLI_ETCD_USERNAME` and `TT_CLI_ETCD_PASSWORD` - config storage connection credentials; apply both to etcd and Tarantool-based storage (already documented)
* `tt install`: `TT_CLI_EE_USERNAME` and `TT_CLI_EE_PASSWORD` - customer zone credentials for installing Tarantool EE
* `tt play`, `tt replicaset`: `TT_CLI_USERNAME` and `TT_CLI_PASSWORD` - same as for `tt connect`
* `tt rocks`: 
  * `TT_CLI_REPO_ROCKS` environment variable value is used as rocks repository path if it is set and
  * `TT_CLI_TARANTOOL_PREFIX` environment variable value is used for as tarantool installation prefix

### PR DESCRIPTION
**Pull request by p7nov**
Resolves #3828 

* Add  authentication section to [tt install ](https://docs.d.tarantool.io/en/doc/gh-3828-tt-env-vars/reference/tooling/tt_cli/install/#authentication)page
* Add authentication section to [tt play](https://docs.d.tarantool.io/en/doc/gh-3828-tt-env-vars/reference/tooling/tt_cli/play/#authentication) page
* Add `TT_CLI_TARANTOOL_PREFIX` and `TT_CLI_REPO_ROCKS` to the [tt configuration page](https://docs.d.tarantool.io/en/doc/gh-3828-tt-env-vars/reference/tooling/tt_cli/configuration/)
* Add `TT_CLI_EE_USERNAME` and `TT_CLI_EE_PASSWORD`, and EE credentials file format to the [tt configuration page](https://docs.d.tarantool.io/en/doc/gh-3828-tt-env-vars/reference/tooling/tt_cli/configuration/)
* Minor fixes on the [tt connect ](https://docs.d.tarantool.io/en/doc/gh-3828-tt-env-vars/reference/tooling/tt_cli/connect/)page

Deployment: https://docs.d.tarantool.io/en/doc/gh-3828-tt-env-vars/reference/tooling/tt_cli/commands/

### CODE REVIEW

**Review by oleg-jukovec — APPROVED** [review]

File: doc/reference/tooling/tt_cli/configuration.rst [hunk 1]
```diff
@@ -116,7 +131,13 @@ ee section
 ~~~~~~~~~~
 
 *   ``credential_path`` -- a path to the file with credentials used for
-    downloading Tarantool Enterprise Edition.
+    downloading Tarantool Enterprise Edition (Tarantool customer zone credentials).
+    The file should contain a username and a password, each on a separate line.
```
Comment (andreyaksenov) [inline doc/reference/tooling/tt_cli/configuration.rst:135]: Maybe, add a link to [Authentication](https://docs.d.tarantool.io/en/doc/gh-3828-tt-env-vars/reference/tooling/tt_cli/install/#authentication)? The example `cred.txt` file looks helpful.

File: doc/reference/tooling/tt_cli/play.rst [hunk 2]
```diff
@@ -57,6 +65,32 @@ on this instance. This means that:
 
 *   Replica IDs will change in accordance with the destination instance configuration.
 
+Authentication
+~~~~~~~~~~~~~~
+
+If authentication is required on the instance, use one of the following ways to pass the
```
Comment (andreyaksenov) [inline doc/reference/tooling/tt_cli/play.rst]: > If authentication is required on the instance

Looks like write access should always require authentication. I'd rephrase this and added a link to https://www.tarantool.io/en/doc/latest/book/admin/access_control/ (there are too many subsystems in Tarantool that require authentication :)).

**Review by andreyaksenov — APPROVED** [review]
